### PR TITLE
limit the occupied memory by syncing block

### PR DESCRIPTION
**Pull request by cyjseagull**
adjust maxRequestBlocks according to the downloading buffer size to fix memory problems

### CODE REVIEW

File: libethcore/Block.h [hunk 1]
```diff
@@ -183,6 +183,7 @@ class Block
     h256 const transactionRoot() { return header().transactionsRoot(); }
     h256 const receiptRoot() { return header().receiptsRoot(); }
 
+    int64_t const& blockSize() const { return m_blockSize; }
```
Comment (bxq2011hust) [inline libethcore/Block.h]: int64_t is unnecessary to use reference

File: libconsensus/pbft/PBFTReqCache.cpp [hunk 2]
```diff
@@ -80,6 +80,31 @@ bool PBFTReqCache::generateAndSetSigList(dev::eth::Block& block, IDXTYPE const&
     return false;
 }
 
+void PBFTReqCache::addViewChangeReq(ViewChangeReq const& req, int64_t const&)
```
Comment (bxq2011hust) [inline libconsensus/pbft/PBFTReqCache.cpp]: if the second parameter is useless, why not delete it ?

Comment (cyjseagull) [reply libconsensus/pbft/PBFTReqCache.cpp]: This will be used in the future to solve consensus-problem

File: libethcore/Block.h (hunk 1 above)
Comment (cyjseagull) [reply libethcore/Block.h]: Fixed.

**Pull request merged**